src/navigator_data_ingest/base/types.py

```diff
@@ -28,6 +38,30 @@ class DocumentType(str, Enum):
     LITIGATION = "Litigation"
 
 
+class UpdateTypes(str, Enum):
+    """Update types supported by the ingest stage."""
+
+    PHYSICAL_DOCUMENT = "PhysicalDocument"
+    FAMILY_DOCUMENT = "FamilyDocument"
+    FAMILY = "Family"
+
+
+class UpdateFields(str, Enum):
+    """Update fields recognized by the ingest stage."""
+
+    SOURCE_URL = "source_url"
+    NAME = "name"
+    DESCRIPTION = "description"
+    DOCUMENT_STATUS = "document_status"
```

Comment:
This looks about right for the moment, but we should talk about dates & how they relate to search...